pilz_psenscan_test_area/execute_test.sh

```diff
@@ -5,20 +5,19 @@ log_ws_info()
   echo "ROS_PACKAGE_PATH=$ROS_PACKAGE_PATH"
 }
 
-launch_psen_scan_node()
+
+test_scan_compare()
 {
-  roslaunch $REPO_NAME psen_scan.launch sensor_ip:=192.168.0.10 host_ip:=192.168.0.20& 2>&1
-  PID=$!
-  echo "Started Process $PID"
+  TEST_FILE="hwtest_scan_compare.test"
+  TEST_ARGS="${TEST_DURATION+test_duration:=$TEST_DURATION} --text --results-base-dir $LOG_FOLDER_NAME"
```

Comment:
I don't quite understand what this section `${TEST_DURATION+test_duration:=$TEST_DURATION}` resolves to be. Is this intended?

Reply:
This is called a parameter expansion. If `TEST_DURATION` is unset (see discussion above) this evaluates to nothing, else it evalutes to `test_duration:=$TEST_DURATION`.